upfadapter/pfcp/udp/pfcp_message_type.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 Canonical Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package udp
+
+import "github.com/wmnsk/go-pfcp/message"
+
+func IsRequest(msg message.Message) bool {
+	switch msg.MessageType() {
+	case message.MsgTypeHeartbeatRequest:
+		return true
+	case message.MsgTypePFDManagementRequest:
+		return true
+	case message.MsgTypeAssociationSetupRequest:
+		return true
+	case message.MsgTypeAssociationUpdateRequest:
+		return true
+	case message.MsgTypeAssociationReleaseRequest:
+		return true
+	case message.MsgTypeNodeReportRequest:
+		return true
+	case message.MsgTypeSessionSetDeletionRequest:
+		return true
+	case message.MsgTypeSessionEstablishmentRequest:
+		return true
+	case message.MsgTypeSessionModificationRequest:
+		return true
+	case message.MsgTypeSessionDeletionRequest:
+		return true
+	case message.MsgTypeSessionReportRequest:
+		return true
```

Comment:
What about something like this? or you can change it to an `if` statement
```suggestion
	case message.MsgTypeHeartbeatRequest, message.MsgTypePFDManagementRequest, message.MsgTypeAssociationSetupRequest, message.MsgTypeAssociationUpdateRequest, message.MsgTypeAssociationReleaseRequest, message.MsgTypeNodeReportRequest, message.MsgTypeSessionSetDeletionRequest message.MsgTypeSessionEstablishmentRequest, message.MsgTypeSessionModificationRequest, message.MsgTypeSessionDeletionRequest, message.MsgTypeSessionReportRequest:
		return true
```

Reply:
Done